kie-server-parent/kie-server-router/kie-server-router-proxy/src/main/java/org/kie/server/router/KieServerRouter.java

```diff
@@ -331,4 +395,16 @@ protected void boostrapFromControllerResponse(String jsonResponse,
 
         adminHandler.addControllerContainers(containers);
     }
+
+    private IdentityService getIdentityService() {
```

Comment:
Would it have more sense to provide FileStoredIdentityManager as default IdentityService and use service loader to discover other implementations?
That would be in line with what we already do in https://github.com/kiegroup/droolsjbpm-integration/blob/master/kie-server-parent/kie-server-controller/kie-server-controller-standalone-impl/src/main/java/org/kie/server/controller/service/StandaloneKieServerControllerImpl.java
Also that way we can get rid of IdentityService.id as whatever user provide will be used, otherwise default will be used.

Reply:
it works like that. you cannot get rid of id, you need a way to differenciate different providers.

Reply:
ok